salt/modules/mac_system.py

```diff
@@ -66,6 +67,87 @@ def _execute_command(cmd, at_time=None):
     return not bool(__salt__['cmd.retcode'](cmd, python_shell=True))
 
 
+def _execute_return_success(cmd):
+    '''
+    Helper function to execute the command
+    Returns: bool
+    '''
+    ret = __salt__['cmd.run_all'](cmd)
+
+    if 'not supported' in ret['stdout'].lower():
+        return 'Not supported on this machine'
+
+    if ret['retcode'] != 0:
+        msg = 'Command Failed: {0}\n'.format(cmd)
+        msg += 'Return Code: {0}\n'.format(ret['retcode'])
+        msg += 'Output: {0}\n'.format(ret['stdout'])
```

Comment:
It would probably be cleaner to combine this into a single format call, rather than three with a bunch of appends.